cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections, OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second")
+                time.sleep(1)

```

Comment:
Using time.sleep(1) is a good way to avoid overloading the system with repeated connection attempts. Good job on this.